code/game/machinery/camera/camera.dm

```diff
@@ -358,6 +357,16 @@
 	busy = 0
 	return 0
 
+/obj/machinery/camera/proc/Togglelight(on=0)
+	for(var/mob/living/silicon/ai/A in player_list)
```

Comment:
If I remember correctly, isn't there an AI list you could use instead? Looping through the player list might be expensive if this proc is called everytime the AI eye moves.